### DISCUSSION TITLE
Access custom REST endpoint?

### DISCUSSION THREAD

**Discussion by mfreeman-xtivia** [lowdefy/lowdefy#1166] [General]
Is it possible to access data from a custom REST endpoint, or does it have to be data from one of the provided connectors?

**Comment by chriseudy** [comment]
I believe you would use AxiosHttp. 
https://docs.lowdefy.com/AxiosHttp
I'm trying to figure that one out myself right now.

**Reply by chriseudy** [reply]
Check this out too. I was missing this last little piece to make this work with a list. 
https://github.com/lowdefy/lowdefy/discussions/655

**Reply by Gervwyk** [reply]
Indeed, AxiosHttp is what you are looking for. Shout if you get stuck :)
